document-message/document-message.bal

```diff
@@ -16,7 +16,9 @@ type ZohoResponse record {|
     |} details;
 |};
 
-final http:Client zohoClient = check new ("http://content.zohoapis.com.balmock.io");
+final http:Client zohoClient = check new ("http://content.zohoapis.com.balmock.io",
+    retryConfig = {count: 3, interval: 1, statusCodes: [404, 408, 500]}
```

Comment:
Let's explain the status codes or remove.

Reply:
Updated